xwayland/server.c

```diff
@@ -107,8 +107,13 @@ noreturn static void exec_xwayland(struct wlr_xwayland_server *server) {
 		dup2(devnull, STDERR_FILENO);
 	}
 
+	const char *xwayland_path = getenv("WLR_XWAYLAND");
```

Comment:
Can we print a debug message when this is used? Since it's a debugging variable, I wouldn't want users to have issues because of a left-over env.

Reply:
Good idea, done.